demisto_sdk/commands/validate/validators/BA_validators/BA125_customer_facing_docs_disallowed_terms.py

```diff
@@ -0,0 +1,65 @@
+
+from __future__ import annotations
+
+from typing import Iterable, List, Union
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.common.files.text_file import TextFile
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Script, Playbook, Pack]
+
+disallowed_terms = (
+        [  # These terms are checked regardless for case (case-insensitive)
+            "test-module",
+            "test module",
+            "long-running-execution",
+        ]
+    )
+
+
+class CustomerFacingDocsDisallowedTermsValidator(BaseValidator[ContentTypes]):
+    error_code = "BA125"
+    description = "Validate that customer facing docs and fields don't contain any internal terms that aren't clear for customers."
+    rationale = "Ensure customer-facing docs avoid internal terms for clarity."
+    error_message = "Found internal terms in a customer-facing documentation file: {terms}"
```

Comment:
add to the error message in which file you found the terms
```suggestion
    error_message = "Found internal terms : {terms} in a customer-facing documentation file: {files}"
```

Reply:
done